coverage.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+mkdir -p api-coverage/C
+mkdir -p api-coverage/Base
+mkdir -p api-coverage/Monad
+
+cd api-coverage
+
+# API documentation
+curl -s https://z3prover.github.io/api/html/group__capi.html | grep -e "Z3_.*" | sed -e "s/\s//g" | grep -oe "Z3_\\w*</a>(" | grep -oe "Z3_\\w*" | sort | uniq > doc-api.txt
+NUM_TOTAL_DOC="$(cat doc-api.txt | wc -l)"
+
+# C API
+cat ../src/Z3/Base/C.hsc | grep "foreign import ccall unsafe" | sed -e "s/foreign import ccall unsafe \"\(.*\)\"/\1/g" | sort | uniq > C-api.txt
+comm -23 doc-api.txt C-api.txt > C/missing.txt
+comm -13 doc-api.txt C-api.txt > C/outdated.txt
+
+function print_coverage {
+	NUM_MISSING=$1
+	NUM_TOTAL=$2
+	NUM_OUTDATED=$3
+
+	COV1=$(python3 -c "print(round((1 - $NUM_MISSING/$NUM_TOTAL_DOC)*100, 2))")
```

Comment:
Not an expert but you could probably use `bc` for the math.

Reply:
Good point about `bc` :+1: 

I already discovered some edge cases that are counted wrong and if they prove to be difficult to fix in Bash I might very well switch to Python.